packages/rxjs/useObservable/index.ts

```diff
@@ -0,0 +1,12 @@
+import { Observable } from 'rxjs'
+import { Ref, ref } from 'vue-demi'
+import { tryOnUnmounted } from '../../core/utils'
+
+export function useObservable<H>(observable: Observable<H>): Readonly<Ref<H>> {
+  const value = ref<H | undefined>()
+  const subscription = observable.subscribe(val => (value.value = val))
```

Comment:
here when i use startWith the subscribe functions should be called immediately but vue still taking undefined as first value

Reply:
here an example https://codepen.io/mdbetancourt/pen/qBZzyQg?editors=0012, "startWith" is printed before "end"

Reply:
maybe have to be with microtask etc, but i'm not familiar with it, any clue?

Reply:
It's working as expected in this: https://codepen.io/antfu/pen/BaKgOYm. Might be some glitchs in your implementation?

Reply:
it's weird here ([https://codepen.io/mdbetancourt/pen/abNebXR](https://codepen.io/mdbetancourt/pen/abNebXR)) works perfectly, maybe is something with the environment (running inside a render function)